### PR TITLE
Added Hotkey for toggling the Crop Setting

### PR DESCRIPTION
**Pull request by jake8**
I noticed that there was no hotkey for toggling the crop option in game, so I decided to take a shot at implementing it myself.

I tried my best to follow your coding style, but please let me know if anything needs to be adjusted. I've tested this on both Linux (Arch) and OS X, and ran the included unit tests.

I believe this also satisfies [this request](https://bugs.dolphin-emu.org/issues/9175]).

### CODE REVIEW

File: Source/Core/DolphinWX/Frame.cpp [hunk 1]
```diff
@@ -1484,6 +1484,11 @@ void CFrame::ParseHotkeys()
 		}
 	}
 
+	if (IsHotkey(HK_TOGGLE_CROP))
+    {
+        g_Config.bCrop = !g_Config.bCrop;
+    }
```
Comment (JosJuice) [inline Source/Core/DolphinWX/Frame.cpp]: You need to use tabs here, not spaces.

File: Source/Core/Core/HotkeyManager.cpp [hunk 2]
```diff
@@ -74,6 +74,8 @@ const std::string hotkey_labels[] =
 	_trans("Decrease Convergence"),
 	_trans("Increase Convergence"),
 
+	_trans("Toggle Crop"),
+
```
Comment (JosJuice) [inline Source/Core/Core/HotkeyManager.cpp]: I think this would fit better before "Toggle Aspect Ratio".

**Comment by jake8** [comment]
Thanks for your feedback! I've made those adjustments.

**Comment by JosJuice** [comment]
Please rebase your two commits into one.

**Comment by jake8** [comment]
Done.

**Comment by JosJuice** [comment]
@dolphin-emu-bot rebuild

**Comment by degasus** [comment]
please don't change the pemissions is those files.

**Comment by jake8** [comment]
I've corrected the permissions to match the other files and rebased the commits into one.

**Pull request merged**